package.json

```diff
@@ -261,8 +261,8 @@
     "regenerator-runtime": "0.13.7",
     "rsuite": "^5.10.0",
     "rxjs": "^7.0.0",
-    "slate": "^0.58.1",
```

Comment:
@tomaszdurka pretty sure slate had a breaking change of format here, or chatgpt gives me the wrong format back :D

Reply:
So you mean this requires quite some migration before merged>

Reply:
I think so, but we prob should test this one beforehand